src/stylesheets/_sub_resource_list.scss

```diff
@@ -0,0 +1,30 @@
+.sub-resource-list {
+    border-top: 1px solid $color-border;
+    margin-top: 1.2rem;
+    margin-bottom: 1.2rem;
+    padding-top: 1.8rem;
+    color: $color-grey-darker;
+    text-align: center;
+
+    @include font-size-text-2;
+
+    .tooltip-trigger {
+        vertical-align: top;
+    }
+
+    .pill {
+        height: 3rem;
+        line-height: 3rem;
+    }
+
+    .more-pill .pill {
+        .more-pill__text {
+            display: inline-block;
```

Comment:
I think using only .more-pill__text here should be enough.

Reply:
updated